Tests/Pcap++Test/Tests/LiveDeviceTests.cpp

```diff
@@ -57,7 +57,7 @@ static bool packetArrivesBlockingModeStartCapture(pcpp::RawPacket* rawPacket, pc
 
 	pcpp::Logger::getInstance().enableLogs();
 
-	int* packetCount = (int*)userCookie;
+	int *packetCount = static_cast<int*>(userCookie);
```

Comment:
nit: move the `*` back next to `int`?
`int*` instead of `int *`

Reply:
Fixed 95e3c97